site/docs/main/config/tls-delegation.md

```diff
@@ -2,7 +2,7 @@
 
 In order to support wildcard certificates, TLS certificates for a `*.somedomain.com`, which are stored in a namespace controlled by the cluster administrator, Contour supports a facility known as TLS Certificate Delegation.
 This facility allows the owner of a TLS certificate to delegate, for the purposes of referencing the TLS certificate, permission to Contour to read the Secret object from another namespace.
-Delegation works for both HTTPProxy and Ingress resources.
+Delegation works for both HTTPProxy and Ingress v1beta1 resources.
```

Comment:
LMK if we should add more detail around Ingress v1 not being supported here, it is mentioned in the Ingress specific page but could be mentioned again

Reply:
I think some more detail would be good, yeah. Something like:
```suggestion
Delegation works for both HTTPProxy and Ingress v1beta1 resources. Ingress v1 is not currently supported due to changes in the spec that make this impossible. See <issue> for details.
```